barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_to_constraint_buf.cpp

```diff
@@ -171,6 +172,11 @@ AcirFormat circuit_serde_to_acir_format(Acir::Circuit const& circuit)
             af.original_opcode_indices.block_constraints.push_back(block.second);
         }
     }
+
+    if (has_brillig) {
+        vinfo("acir_format:circuit_serde_to_acir_format: Encountered unhadled BrilligCall during circuit "
```

Comment:
```suggestion
        vinfo("acir_format:circuit_serde_to_acir_format: Encountered unhandled BrilligCall during circuit "
```

Having brillig opcodes in the ACIR is expected however. I'm not sure why we'd need to emit a log for this.

Reply:
Ah, this is just avoiding log spam for every brillig call. I think we can remove this entirely however though.